package.json

```diff
@@ -22,7 +22,7 @@
     "express": "^4.16.3",
     "js-yaml": "^3.12.0",
     "percy-client": "^3.0.1",
-    "puppeteer": "^1.7.0",
+    "puppeteer": "^1.10.0",
     "retry-axios": "^0.3.2",
     "sinon": "^6.1.5",
```

Comment:
Unrelated, but should this be a dependency?

Reply:
Nope! Feel free to issue a PR for that @Robdel12. Good find.